Project.toml

```diff
@@ -3,6 +3,7 @@ uuid = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 version = "0.9.8"
 
 [deps]
+HypergeometricFunctions = "34004b35-14d8-5ef3-9330-4cdb6864b03a"
```

Comment:
This requires a `[compat]` entry.

This also pulls in some additional dependencies since HypergeometricFunctions depends on DualNumbers, it seems? I am just wondering if people are fine with this since my ChainRules PR (https://github.com/JuliaStats/StatsFuns.jl/pull/106) was not received positively mainly due to the additional dependency on ChainRulesCore.

On a side note, HypergeometricFunctions would be needed in SpecialFunctions to resolve https://github.com/JuliaMath/SpecialFunctions.jl/issues/317 and https://github.com/JuliaMath/SpecialFunctions.jl/issues/160. There was some discussion about depending on HypergeometricFunctions in https://github.com/JuliaMath/SpecialFunctions.jl/pull/175, but I assume for the very same reasons not everyone would welcome such a dependency in SpecialFunctions.

Reply:
I'll add a compat entry (or CompatHelper will do it).

I can try to measure the effect on load time which I assume is the main concern here. I doubt that it will be much and more generally, I think that HypergeomericFunctions is a natural dependency here. Many CDFs have hypergeometric representations so I think this is a bit different from the AD discussion. I don't think it would make sense to have the CDFs defined in a separate package. (I think I'm in favor of #106 btw)

Reply:
I agree that probably it is more central to the package than AD (even though, of course, I'm in favor of the ChainRules PR :wink: ). I am fine with the dependency but I wanted to raise it due to the reactions in the other PR.